LabGui.py

```diff
@@ -804,6 +807,18 @@ def add_widget(self, widget_creation, action_functions=None, **kwargs):
         """
         widget_creation(self)
 
+    def get_ren(self):
+        # This returns default ren state from corewidget
+        return self.widgets['RENWidget'].get_ren()
+
+    def set_ren(self):
+        self.widgets['RENWidget'].set_ren(IOTool.get_REN())
+
+    def save_ren(self):
+        level = self.widgets['RENWidget'].get_ren()
+        curr = IOTool.get_REN()
+        if level is not None or curr is not None:
+            IOTool.set_REN(level)
```

Comment:
The philosophy we had with @ben741, was to write as little Widget code in `LabGui.py` as possible. These functions could be defined in the `RemoteEnableWidget.py` `add_widget_into_main` in the same way I did it here : 
https://github.com/LabGUI/LabGUI/blob/f34e0e73f5b9857f5fbbfb68cc4101a5c4b859f2/LabTools/CoreWidgets/LoadPlotWidget.py#L291

Reply:
That way everything which has to do with the widget is located in a single file

Reply:
corrected, and tested solely for intended functionality of the functions